src/main/kotlin/sliv/tool/view/HelpView.kt

```diff
@@ -0,0 +1,5 @@
+package sliv.tool.view
+
+class HelpView : Dialog("Help") {
```

Comment:
empty class

Reply:
Empty views matches with such classes on the components diagram. It is not empty, because it passes parameter into the parent constructor